src/main/webapp/dev/js/main/instructorFeedbackEdit.js

```diff
@@ -302,6 +309,7 @@ function bindFeedbackSessionEditFormSubmission() {
  * @param questionNum
  */
 function disableQuestion(questionNum) {
+    $(`#${ParamsNames.FEEDBACK_QUESTION_EDITTYPE}-${questionNum}`).val('edit');
```

Comment:
Hmm? What is the purpose of changing the edit type to `edit` here? Was your intention to make the state as consistent as possible?

But we can't account for new functions and I'm not sure if we have missed any other cases where it should have been `edit` rather than `delete`.

That's why it's easier to just set the edit type to `edit` when submitting.